src/binary/Y2RubyComponent.cc

```diff
@@ -75,16 +76,30 @@ Y2Namespace *Y2RubyComponent::import (const char* name)
   args->add (YCPString(/*module*/ name));
   args->add (YCPString(/*module*/ module));
 
-  try {
-    YRuby::loadModule (args);
+  if (YRuby::loadModule (args))
+  {
     y2debug("Module '%s' loaded", name);
     // introspect, create data structures for the interpreter
     Y2Namespace * res = new YRubyNamespace (name);
     namespaces[name] = res;
     return res;
-  } catch (exception& e) {
```

Comment:
You remove a `catch` but remove no `throw`. This will fail in unexpected ways I think.